### PR TITLE
Add ability to change password of users via StaffCP

### LINKED ISSUE #3093: Add set password field to userCP
### Describe the feature

Because some people do not want to setup phpmailer, I think it would be useful to allow administrators to set a user their password manually to what they want. This way they can set it to something they can share with the person who forgot their password and then those people can change their password again. 

Currently when a user forgets their password, if phpmailer is not setup there is no way to reset the user their password as far as I'm aware.

### PR DESCRIPTION
**Pull request by supercrafter100**
Closed #3093 

Preview:
image

### CODE REVIEW

**Comment by tadhgboyle** [comment]
What is the red box?

We also def want a confirmation password input too. Could these two inputs be on their own floating container? It looks a little wonky to just have them thrown at the bottom

**Comment by supercrafter100** [comment]
> What is the red box?
> 
> We also def want a confirmation password input too. Could these two inputs be on their own floating container? It looks a little wonky to just have them thrown at the bottom

Red box is a browser password extension, nothing important. I'll make them on their own floating container

**Comment by supercrafter100** [comment]
New workflow: image

**Comment by partydragen** [comment]
Amazing good work :D

File: modules/Core/pages/panel/users_edit.php [hunk 1]
```diff
@@ -210,19 +210,42 @@
...
+
+            if ($validation->passed()) {
+                $password = Input::get('password');
+                $encrypted_password = password_hash($password, PASSWORD_BCRYPT, ['cost' => 13]);
+                $view_user->update([
+                    'password' => $encrypted_password
+                ]);
+                Session::flash('edit_user_success', $language->get('admin', 'user_password_changed_successfully'));
+            } else {
+                Session::flash('edit_user_error', implode('\n', $validation->errors()));
```
Comment (tadhgboyle) [inline modules/Core/pages/panel/users_edit.php:248]: Hmm, this feels really dirty. Not sure there is anything else we can do here though... Revamping how we handle success/error messages to simplify everything would be nice eventually:tm:

Comment (supercrafter100) [reply modules/Core/pages/panel/users_edit.php:248]: I can't really think of a better way to do this currently

File: modules/Core/pages/panel/users_edit.php [hunk 2]
```diff
@@ -210,19 +210,42 @@
...
-                if ($user_query->id > 1) {
-                    EventHandler::executeEvent('deleteUser', [
-                        'user_id' => $user_query->id,
-                        'username' => $user_query->username,
-                        'email_address' => $user_query->email
-                    ]);
 
-                    Session::flash('users_session', $language->get('admin', 'user_deleted'));
-                }
+        } else if ((Input::get('action') == 'delete') && $user_query->id > 1) {
```
Comment (Aberdeener) [inline modules/Core/pages/panel/users_edit.php:214]:
```suggestion
        } else if (Input::get('action') == 'delete' && $user_query->id > 1) {
```

**Review by tadhgboyle — APPROVED** [review]

**Review by partydragen — APPROVED** [review]

**Pull request merged**